include/libtorrent/alert_types.hpp

```diff
@@ -2066,6 +2067,11 @@ namespace libtorrent
 		boost::array<char, 64> signature;
 		std::string salt;
 		boost::uint64_t seq;
+
+		// DHT put operation usually writes item to k nodes, maybe the node
+		// is stale so no response, or the node doesn't support 'put', or the
+		// token for write is out of date, etc.
```

Comment:
I suppose this is supposed to count the number of successful responses we got from the puts. (but the comment doesn't actually spell that out). Btw, do you know if we keep trying more nodes, farther away from the target, if puts fail? to make a best-effort attempt at storing the blob on at least k nodes.